packages/editor/src/components/browser/cms/CmsBrowser.tsx

```diff
@@ -155,10 +173,7 @@ const CmsBrowser = ({ value, onChange, noApiCall, typeFilter, onDoubleClick, loc
 
   return (
     <>
-      <div className='browser-table-header'>
-        <Checkbox label='Enable required Projects' value={requiredProject} onChange={() => setRequiredProject(!requiredProject)} />
-        <Button onClick={() => refetch()} title='Refresh CMS-Browser' aria-label='refresh' icon={IvyIcons.Redo} />
-      </div>
+      <Checkbox label='Enable required Projects' value={requiredProject} onChange={() => setRequiredProject(!requiredProject)} />
```

Comment:
what happens if you select a node from a different project as parent for the new cms entry? 
I think we need this new cms thing only on the own project, so we should then disable the button for others